source/assets/javascripts/common.js

```diff
@@ -76,5 +90,32 @@ $(function () {
     $(".top-bar").removeClass('top-bar--show');
   });
 
-  $('[data-toggle="tooltip"]').tooltip()
+  $('[data-toggle="tooltip"]').tooltip();
+
+  //Slick Slider
+  $('.carousel-list').slick({
```

Comment:
Is this required? There's another file that takes care of slick, why not move this code in there?

Reply:
You're right, we can call this js only on home page

Reply:
Does it import, manually? Seems that the imports from packages work only if the `import` is on the `common.js` file.

Reply:
We could delete the `NoUiSlider` that is about the _price calculator_ that we had on the previous version of the home page.

Reply:
I meant that there's a file called `source/assets/javascripts/slickSlider.js`, maybe we can move this slick call in there? Or is there a reason I'm missing to keep them separate?

Reply:
Ok, now I understand! I delete it!